Makefile

```diff
@@ -0,0 +1,72 @@
+clean:
+	rm -rf bazel-*
```

Comment:
This does not actually remove thigns (the bazel-* things are symbolic links to its internal cache).
But `bazel clean` does.